_layouts/post.html

```diff
@@ -44,3 +47,20 @@ <h3>
 </div>
 {% endif %}
 {% include tagcloud.html %}
+
+<script>
+  let i = 0;
+  const text = '{{ page.description }}';
+  const speed = 50;
```

Comment:
@tildehacker can we somehow make this configurable via the `_config.yml`?

Reply:
Yes, I added a new variable called `animation_speed` in `_config.yml` to make it configurable.